src/praisonai/praisonai/cli/commands/flow.py

```
@@ -126,6 +126,292 @@ def flow_start(
+        
+        # Upload to Langflow
+        from praisonai.flow.client import create_client
+        
+        console.print(f"[cyan]Connecting to Langflow at {langflow_url}...[/cyan]")
+        client = create_client(langflow_url)
+        
+        # Check server health
+        health = client.health_check()
+        if health["status"] != "healthy":
+            console.print(f"[red]Error: Langflow server not accessible at {langflow_url}[/red]")
+            console.print(f"[yellow]Make sure Langflow is running: praisonai flow[/yellow]")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)